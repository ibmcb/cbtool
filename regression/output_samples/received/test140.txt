Usage: airunstate <cloud_name> <identifier> <runstate> [command = unknown] [firs = none] [mode] 
