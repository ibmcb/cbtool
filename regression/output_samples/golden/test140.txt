Usage: aishow <cloud_name> <identifier> [key = all] 
